### ISSUE TITLE
Add custom permission for specific group type

### ISSUE THREAD

**Issue by Allan-Nava** [django-guardian/django-guardian#538]
Hi @ad-m ,
I use [django-groups-manager]( https://github.com/vittoriozamboni/django-groups-manager), Is it possible to give group type permissions for the belonging group?

For example:
I create Inter FC and I need to have a group("society" group) who handle the permission to add new, remove users.
In the society group, I add the society's member like Zhang Jindong and Moratti, they have these permission only for INTER FC.

I added in my admin.py
```
#GroupTypeAdmin
class GroupTypeAdmin(GuardedModelAdmin):

    pass

#admin.site.register(models.Group)
admin.site.register(Group, GroupAdmin)
admin.site.register(Member)
admin.site.register(GroupType, GroupTypeAdmin)
```

But i got only for user this:
![screen shot 2017-12-04 at 16 13 32](https://user-images.githubusercontent.com/22498435/33559758-225cb0e6-d90e-11e7-9425-b8c48c21c8b0.png)

**Comment by ad-m** [comment]
I have no idea and I do not have time to analyze third-party library.

**Comment by Allan-Nava** [comment]
Ok thanks, but is possible add permission to a group? @ad-m 

I think is the same issue #532

**Comment by ad-m** [comment]
Assigning permissions to groups is supported, as well as assigning user permissions. In some cases, checking user permissions through groups can be disabled by default because of performance, but you can run them.

**Comment by Allan-Nava** [comment]
Ok, but I need in the admin Panel such as this issued closed #536
